_includes/page__hero_video.html

```diff
@@ -0,0 +1,17 @@
+{% include base_path %}
+
+{% capture video_id %}{{ page.header.video.id }}{% endcapture %}
+{% capture video_provider %}{{ page.header.video.provider }}{% endcapture %}
+
+<div class="page__hero{% if page.header.overlay_color or page.header.overlay_video_path %}--overlay{% endif %}"
```

Comment:
What is the purpose of this code? As a test I added the following YAML Front Matter to add a color overlay as the code seems to suggest is possible, and no video was embedded.

```yaml
header:
  video:
    provider: "youtube"
    id: "gOBj8HdfA2Y"
  overlay_color: "#333"
```

Had to remove `overlay_color` for it to work.